docs/gemstones/containers/podman.md

```diff
@@ -98,8 +99,9 @@ Other file types are supported: pod, volume, network, image and kube. [Pods](htt
 
 Podman additionally provides the `generate systemd` subcommand. It can be used to generate `systemd` service files. 
 
-> [!IMPORTANT]
-> `generate systemd` is now deprecated and will not receive further features. Usage of Quadlet is recommendet.
+!!! warning
```

Comment:
```suggestion
!!! warning "Warning"
```

Reply:
I prefer this syntax to make the translation work easier
(feel free to modify or revert my changes)

Reply:
I trust your experience on that one… And I do not have any opinion on that :D